ddp-workspace/projects/ddp-sdk/src/lib/components/address/addressEmbedded.component.ts

```diff
@@ -182,6 +182,14 @@ export class AddressEmbeddedComponent implements OnDestroy, OnInit {
      */
     @Output()
     public valueChanged = new EventEmitter<Address | null>();
+
+    /**
+     * Will emit an address whenever it is updated from ddp-address-input component
+     * type {EventEmitter<Address>}
+     */
+    @Output()
+    public inputAddress = new EventEmitter<Address | null>();
```

Comment:
The address that comes from `ddp-address-input` component is meant to remain private within the embedded address. Let's not externalize this. It is not a clean address/not meant to be seen outside of addressEmbedded.